TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+// This class makes a pair of color for major and minor color.
+{   class ColorPair
+    {   private Color majorColor;
+        public Color MajorColor
+        { get { return majorColor; } set { majorColor = value; } }
+        private Color minorColor;
+        public Color MinorColor

```

Comment:
you can remove the definition get set everything with the below single line

internal Color Major color {internal get; private set;}